lib/install/management/create.go

```diff
@@ -60,6 +62,16 @@ func (d *Dispatcher) CreateVCH(conf *config.VirtualContainerHostConfigSpec, sett
 		return errors.Errorf("Creating the appliance failed with %s. Exiting...", err)
 	}
 
+	// send the signal to VCH logger to indicate VCH datastore path is ready
+	datastoreReadySignal := vchlog.DatastoreReadySignal{
+		Datastore: d.session.Datastore,
+		LogFileName: "vic-machine-create",
+		Operation: trace.NewOperation(d.ctx, "vic-machine create"),
```

Comment:
didn't see much value for the operation ID here, because you're writing logs to different files for different operation in different time.

Reply:
there could be more than one vic machine processes at the same time, to prevent overwriting, we also included the operation ID together with the timestamp